app/scripts/controllers/home.ctrl.js

```diff
@@ -125,7 +139,28 @@ angular.module('yourStlCourts').controller('HomeCtrl', function (Citations,toast
     modalInstance.result.then(function (selectedMunicipalities) {
       ctrl.citationCriteria.municipalityNames = selectedMunicipalities;
     });
+
   };
 
+  function modifyMunicipalities(municip){
```

Comment:
Is this logic going to be used throughout the app? If so, bury it behind the Municipalities service and just always return the list from the promise with this change. Also - it looks like this is duplicated in your MuniObj I commented on below.